recipes-sota/aktualizr/aktualizr-shared-prov-creds.bb

```diff
@@ -9,18 +9,48 @@ inherit allarch
 DEPENDS = "aktualizr-native zip-native"
 ALLOW_EMPTY_${PN} = "1"
 
+T = "${WORKDIR}/tmp"
```

Comment:
From the manual (section linked by Patrick), it sounds like this is already defined and you do not need to redefine it yourself. Does it work without?

Reply:
Yes, it works without.